src/lib.rs

```diff
@@ -225,7 +363,34 @@ impl BitAnd for Integer {
 
```

Comment:
It should be `&` instead of `mod_floor`.

Reply:
Thanks, good catch.